server/routes/email.js

```diff
@@ -0,0 +1,59 @@
+require("dotenv").config();
+const express = require("express");
+const router = express.Router();
+const sendGridApiKey = process.env.SENDGRID_API_KEY;
+const sgMail = require("@sendgrid/mail");
+const auth = require("../middleware/auth");
+
+sgMail.setApiKey(sendGridApiKey);
+
+const sendEmail = async (req, res) => {
+  //Fake Flight Data to test the email
+  const flightData = {
+    passenger: "teamAngelFish",
+    from: "Toronto",
+    to: "Bali",
+    departureTime: "12PM",
+    arrivalTime: "1PM",
+    hotelName: "La Crop",
+  };
+
+  const {
+    passenger,
+    from,
+    to,
+    departureTime,
+    arrivalTime,
+    hotelName,
+  } = flightData;
+
+  let msg = {
+    //  Chnage this to your own email to get mail sent to your inbox
+    to: "gbudjeakp@gmail.com",
+
+    //Do not change verified (The from email) Sender as SendGrid will only send from this email.
+    from: "sebdwebdev@gmail.com",
```

Comment:
You probably don't want this on GitHub...

Reply:
I'll get rid of that ASAP.